functions-and-scope.js

```diff
@@ -43,6 +66,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // ---- Verwachte uitkomst: 6.642857142857143
 
 
+const arrayLength = grades.length;
+let teller2 = 0;
+for ( let i = 0; i < grades.length; i++){
+    teller2 = teller2 + grades[i];
```

Comment:
Dit kun je wat korter schrijven
```suggestion
    teller2 += grades[i];
```